### DISCUSSION TITLE
How do I configure endpoint filtering?

### DISCUSSION THREAD

**Discussion by lovexian** [SkyAPM/SkyAPM-dotnet#599] [Q&A]
### Question
How do I configure endpoint filtering?
I need to filter out some endpoints such as heartbeat detection, but I can't find documentation for .net

**Comment by wu-sheng** [comment]
How to filter? You have a specific endpoint list?

**Reply by lovexian** [reply]
是的吴总,就是端点过滤,需要过滤特定的一些端点,比例心跳检测那些没必要追踪的,看到java的agent是有一个apm-trace-ignore-plugin-x.jar包, .net这边是如何处理?我没找到相关文档

**Comment by lovexian** [comment]
我试图配置在这里,但是没生效
  "Sampling": {
      "SamplePer3Secs": -1,
      "Percentage": -1.0,
      "IgnorePaths ":["/health","http://consul.default:8500/*"]
    },

**Comment by wu-sheng** [comment]
Please use English on the GitHub.
